lib/S3Plugin.js

```diff
@@ -259,15 +263,19 @@ class S3Plugin {
 
    * Get list of files keys from S3 bucket.
    */
-  async getFilesKeys() {
+  async getFilesKeys(request) {
     this._assertCredentials();
+
+    const Bucket = this._stringArg(request, 'bucketName', this.config.bucketName);
```

Comment:
Variable should not start with a capital letter

Reply:
done renamed it bucketName has done everywhere in plugin